schema/annotation-source.yaml

```diff
@@ -185,7 +190,23 @@ $defs:
         enum: [ "supports", "uncertain", "opposes" ]
         description: >-
           The direction of this statement with respect to the target proposition.
-
+      has_evidence_of_type: 
+        type: string
+        enum: [ "case-control evidence", "clinical testing evidence", "curation-based evidence", 
```

Comment:
Here, we would cast ClinVar's enumerated values for collection method as evidence types (which IMO is an equally valid way to consider them).

Reply:
I won't be able to use this, because, like the "allele_origin" field in clinvar, the collection method can have combinations of values that span a wide array of choices. So, I will not be using "has_evidence_types" initially. 

That said, the collection_method and allele_origin are two values that the users are interested in, so maybe they can write up a policy on how I can transform the clinvar data into this normalized field. If so, I will be glad to use it.

Reply:
The `has_evidence_of_type` attribute  can take multiple values - so it's not a problem if ClinVar provides more than one.